op-supervisor/supervisor/backend/depset/rollup_config_set.go

```diff
@@ -0,0 +1,143 @@
+package depset
+
+import (
+	"context"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+)
+
+type RollupConfigSetSource interface {
+	LoadRollupConfigSet(ctx context.Context) (RollupConfigSet, error)
+}
+
+// RollupConfigSet provides access to minimal rollup configuration for a set of chains.
+// Implementations should panic if any of the methods, besides HasChain, are called for a chain
+// that is not part of the rollup config set.
+type RollupConfigSet interface {
+	HasChain(chainID eth.ChainID) bool
+	Chains() []eth.ChainID
+	Genesis(chainID eth.ChainID) Genesis
+	IsInterop(chainID eth.ChainID, ts uint64) bool
+	IsInteropActivationBlock(chainID eth.ChainID, ts uint64) bool
+}
+
+type StaticRollupConfigSet map[eth.ChainID]*StaticRollupConfig
+
+// StaticRollupConfig provides the rollup information relevant for Interop.
+// It's a trimmed down version of [rollup.Config].
+type StaticRollupConfig struct {
+	// Genesis anchor point of the rollup
+	Genesis Genesis `json:"genesis"`
```

Comment:
maybe
```suggestion
	Genesis types.DerivedBlockSealPair `json:"genesis"`
```
but maybe too internal